src/runtime/styles/_mixins.scss

```diff
@@ -0,0 +1,84 @@
+$chevronIcon: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="none" viewBox="0 0 16 16"><path fill="currentColor" fill-rule="evenodd" d="M2.97 5.47a.75.75 0 0 1 1.06 0L8 9.44l3.97-3.97a.75.75 0 1 1 1.06 1.06l-4.5 4.5a.75.75 0 0 1-1.06 0l-4.5-4.5a.75.75 0 0 1 0-1.06" clip-rule="evenodd"/></svg>');
+
+@mixin chevron() {
+    position: relative;
+
+    padding-left: 28px;
```

Comment:
All values are multiples of 4, maybe use css variables like in gravity?

https://github.com/gravity-ui/uikit/blob/05315c8fef7b45c7c41464468f7716d6f191aab5/styles/themes/common/spacing.scss#L9

Reply:
It would make sense if these variables were set in the basic diplodoc/transform (`.yfm`) styles.